base/sparse/linalg.jl

```diff
@@ -499,7 +499,196 @@ function norm(A::SparseMatrixCSC,p::Real=2)
     throw(ArgumentError("invalid p-norm p=$p. Valid: 1, Inf"))
 end
 
-# TODO
+# TODO rank
+
+# cond
+function cond(A::SparseMatrixCSC, p::Real=1)
```

Comment:
`cond` defaulting to different norms for sparse vs dense is a little iffy.

And I think `normestinv` would look a little nicer without the leading and trailing lines of whitespace inside the method - the spacing within the code is fine, but right after `function` and right before `end` doesn't serve much purpose.